### DISCUSSION TITLE
Pls elaborate more on what "Comfortable w Jas and Vin ABDL" entails on the prompt

### DISCUSSION THREAD

**Discussion by xl3lackout** [zippity21/Stardew_Valley_Regression_Mod#83] [General]
Its a niche complaint, but considering how varied the aims of a lot ABDLs can be, idk if its like, treating me like im the same age, maybe some jodi CG, grade school headspace type stuff. Or if its on the more NSFW end. Sorry if it seems like a dumb or repugnant question, but I think the former is wholesome, but I aint comfortable with the latter at all, and I think some specification onto what kind of content it enables would put some peoples minds at ease, mine included.

Edit: I posted this like right after i got the prompt after just playing the mod for the first time. Sorry if it came off as distressed, cuz I wasn't sure at all what to expect. It ended up being what i was hoping for and ive really been enjoying it. Its still worth noting if the prompt should say that enabling it causes them to treat you more like a fellow kid overtime, and maybe that it discusses their potty habits, just for the sake of clarity. But I dont think I projected my feelings the best, sorry

**Comment by zippity21** [comment]
Probably a good thing to put in the README. Where I have had no personal time in about a year (see 8 month gap in updates), I'm going to label this as a documentation request so I don't forget about it.

**Comment by TikkuTokku** [comment]
I'm the one that wrote that prompt, and you're right, it can be reworked to be a little more clear. I gave it a quick little edit to:

"The kids can have dialogue about being in diapers and reacting to yours. Would you be comfortable with that?"

and 

"The more hearts you have with any villager the more diaper related dialogue there will be."

Does that clear things up enough?

**Reply by Gooberology** [reply]
Yeah, I ended up finding the dialogue really endearing, I was just a little fearful at the ambiguity of what i was getting myself into. Im used to ABDL content being very heavy handed and blatantly sexual.
